index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <main>
+        
+            <nav>
+                <div class="logo">
+                    <p>My Logo</p>
```

Comment:
- [ ] Please use anchor tag `<a>` for the logo. Logos are supposed to be links so the website users can navigate to the home page by clicking on the logo.

Reply:
I have enclosed an anchor tag inside a paragraph tag. Hope that works. This is so that it can align center on the vertical axis